[2023-08-31T06:30:01.471+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1 permission to 509
[2023-08-31T06:30:01.472+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1'
[2023-08-31T06:30:01.546+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1 permission to 509
[2023-08-31T06:30:01.547+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1'
[2023-08-31T06:30:01.725+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_python_operator.py_t1 scheduled__2023-08-30T06:30:00+00:00 [queued]>
[2023-08-31T06:30:01.733+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_python_operator.py_t1 scheduled__2023-08-30T06:30:00+00:00 [queued]>
[2023-08-31T06:30:01.734+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-31T06:30:01.775+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): py_t1> on 2023-08-30 06:30:00+00:00
[2023-08-31T06:30:01.781+0000] {standard_task_runner.py:57} INFO - Started process 4317 to run task
[2023-08-31T06:30:01.785+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_python_operator', 'py_t1', 'scheduled__2023-08-30T06:30:00+00:00', '--job-id', '218', '--raw', '--subdir', 'DAGS_FOLDER/dags_python_operator.py', '--cfg-path', '/tmp/tmphn1wtaz6']
[2023-08-31T06:30:01.787+0000] {standard_task_runner.py:85} INFO - Job 218: Subtask py_t1
[2023-08-31T06:30:01.855+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1 permission to 509
[2023-08-31T06:30:01.856+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_operator/run_id=scheduled__2023-08-30T06:30:00+00:00/task_id=py_t1'
[2023-08-31T06:30:01.859+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_python_operator.py_t1 scheduled__2023-08-30T06:30:00+00:00 [running]> on host 07fcf706bef2
[2023-08-31T06:30:01.993+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_python_operator' AIRFLOW_CTX_TASK_ID='py_t1' AIRFLOW_CTX_EXECUTION_DATE='2023-08-30T06:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-30T06:30:00+00:00'
[2023-08-31T06:30:01.996+0000] {logging_mixin.py:150} INFO - Pomeranian
[2023-08-31T06:30:01.996+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-08-31T06:30:02.009+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dags_python_operator, task_id=py_t1, execution_date=20230830T063000, start_date=20230831T063001, end_date=20230831T063002
[2023-08-31T06:30:02.077+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-31T06:30:02.099+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
